Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -1832,6 +1844,31 @@ public enum NIOHTTP2Errors {
         }
     }
 
+    /// The remote peer has triggered too many stream errors on this connection.
```

Comment:
We might broaden the definition here in future, so let's just call it `glitches` instead of `stream errors`.